### DISCUSSION TITLE
Show ELNs and other info with `ls` or after `clear`

### DISCUSSION THREAD

**Discussion by Slackadays** [leo-arch/clifm#206] [Ideas]
If I use `clear` to clear the screen, **clifm**'s prompt is now at the top, like it should be. However, the listing of all the files is missing. A typical user might want to do `ls` to bring them all up again, but this doesn't work as it just does `ls` without any of **clifm**'s special sauce. Therefore, I think it would be a great UX addition to replace `ls` with the **clifm** directory listing instead. Alternatively, you could also show the listing even after `clear` clears the screen.

**Comment by leo-arch** [comment]
[Aliases](https://github.com/leo-arch/clifm/wiki/Customization#aliases) are your best friend here.

Just create a few aliases like these:

```sh
alias ls='rf'
alias clear='rf'
```
Customize these commands to fit your needs.

**Reply by leo-arch** [reply]
Hey @Slackadays, is this approach good enough?

**Reply by Slackadays** [reply]
@leo-arch I'm using the `rf` command now as a replacement for `ls` when I'm using clifm. Personally, I hate having to make aliases and that's why I don't have any at all for my terminal. Maybe a different approach could be to prioritize the `rf` command in the documentation because I only just found out about it once you provided the alias examples.

**Reply by leo-arch** [reply]
Great. I'll try to highlight `rf` a bit more in the docs.

EDIT: <kbd>Enter</kbd> (on empty line) and <kbd>Ctrl-l</kbd> do the same thing as the `rf` command.
